### DISCUSSION TITLE
After upgrading to macOS Ventura Cypress component testing has stopped working.

### DISCUSSION THREAD

**Discussion by mschaefer-gresham** [cypress-io/cypress#24868] [Component Testing]
After upgrading to Ventura, Cypress component testing has stopped working. I've seen other related posts. I've tried granting Full Disk Access and App Management rights to every program involved: iTerm, Webstorm, yarn, npm, node, Cypress. 

Cypress hangs here (```yarn cypress open --component```):

<img width="634" alt="image" src="https://user-images.githubusercontent.com/94627216/204473592-cabc4f29-cec9-4a94-9ad8-7ab4c1f8ac96.png">

I had node 16 installed before upgrading to Ventura. With node 16, ```yarn cypress run --component --browser chrome``` still works. But ```yarn cypress open --component``` hangs as above. I installed node 19 after upgrading to Ventura and both commands hang using version 19. 

Using node 19, ```yarn cypress run --component --browser chrome``` hangs here on the first test:

<img width="828" alt="image" src="https://user-images.githubusercontent.com/94627216/204481533-7b55bf0d-f2db-4973-9835-8de8cca84f6e.png">

Another member on my team only had to give App Management rights to iTerm and Webstorm and Cypress is working. But he didn't install a new node version after upgrading or make any other changes. I had also freshly cloned our repo and did a yarn install after the upgrade which he didn't do. I'm guessing that if he made similar changes he would have the same issues.

If I run ```sudo yarn cypress open --component``` the tests load, but when I try to run one, it just hangs here:

<img width="587" alt="image" src="https://user-images.githubusercontent.com/94627216/204557148-55e2cc07-34df-4ea8-93fc-8c674f15d23c.png">

I tried givng iTerm Full Disk Access, App Management rights, and added it to Developer Tools (all under Privacy and Security). 

This is frustrating. Any suggestions would be welcome.

**Comment by emilyrohrbough** [comment]
@mschaefer-gresham Sorry to hear you are having issues. Could please open an issue with these details?

**Reply by mschaefer-gresham** [reply]
I just opened [24876](https://github.com/cypress-io/cypress/issues/24876)
